boudicca.base/enricher-openapi/build.gradle.kts

```diff
@@ -35,15 +33,40 @@ dependencies {
     api(project(":boudicca.base:semantic-conventions"))
 }
 
-tasks.withType<org.openapitools.generator.gradle.plugin.tasks.GenerateTask> {
+tasks.withType<GenerateTask> {
+    inputs.files(openapi)
+    inputSpec.set(openapi.files.first().path)
+}
+
+tasks.register<GenerateTask>("generateJavaClient") {
     inputs.files(openapi)
     inputSpec.set(openapi.files.first().path)
+    outputDir.set(layout.buildDirectory.dir("/generated/java").get().toString())
```

Comment:
i think this is interpreted as an absolute path, so remove the first / or try ./generated